CleverPush/CleverPush/Source/CleverPushInstance.m

```diff
@@ -1206,6 +1209,10 @@ - (void)setSyncSubscriptionRequestData:(NSMutableURLRequest*)request {
             [dataDic setObject:@"1" forKey:@"topicsVersion"];
         }
     }
+    
+    if ([[channelConfig valueForKey:@"preventDuplicatePushesEnabled"] boolValue] == YES) {
```

Comment:
please use `boolForKey` instead of `valueForKey` it will handle nil values. Then we also do not need boolValue